internal/webhook/external/github.go

```diff
@@ -166,13 +167,17 @@ func (g *githubWebhookReceiver) getHandler(requestBody []byte) http.HandlerFunc
 				xhttp.WriteErrorJSON(w, err)
 				return
 			}
-			manifest := pkg.GetPackageVersion().GetContainerMetadata().GetManifest()
+			v := pkg.GetPackageVersion()
+			manifest := v.GetContainerMetadata().GetManifest()
+			// Determine if the package is a Helm chart
 			if cfg, ok := manifest["config"].(map[string]any); ok {
 				if mediaType, ok := cfg["media_type"].(string); ok {
 					repoURL = normalizeOCIRepoURL(ref.Context().Name(), mediaType)
 				}
+			} else {
+				repoURL = image.NormalizeURL(ref.Context().Name())
 			}
-
+			qualifier = v.GetVersion()
```

Comment:
Looking at actual payloads, it seems this may be incorrect.

This seems to be the right way:

```suggestion
			qualifier = v.GetContainerMetadata().GetTag().GetName()
```